src/commands/autocomplete/script.js

```diff
@@ -48,7 +49,8 @@ compinit;`)
       //   fs.copySync(cli, client)
       // }
       const plugins = await new Plugins(out).list()
-      const cmds = plugins.map(p => p.commands.filter(c => !c.hidden).map(c => {
+      const cmds = plugins.map(p => p.commands.filter(c => !c.hidden || !!c.id).map(Command => {
+        const c = typeof Command === 'function' ? Command : convertFromV5((Command: any))
```

Comment:
I feel that this file should not have to be aware of v5 commands. Can you instead make a helper inside of `plugins/index.js` to export all the commands already converted to v6?